lib/ansible_tower_client/api2.rb

```diff
@@ -0,0 +1,7 @@
+module AnsibleTowerClient
+  class Api2 < Api
+    def credential_class
+      @credential_class ||= AnsibleTowerClient::CredentialV2
```

Comment:
Any reason we can't switch this like the other types?
https://github.com/ansible/ansible_tower_client_ruby/blob/master/lib/ansible_tower_client/api.rb#L182